app/models/post.rb

```diff
@@ -1,5 +1,15 @@
 class Post < ApplicationRecord
-  belongs_to :user
+  after_save :update_postscounter
+
+  belongs_to :author, class_name: 'User'
   has_many :comments
-  has_many :likes
+  has_many :posts
+
```

Comment:
- [ ] Great job so far 👍🏻 however you've made a mistake here 🙃  `Post has_many posts` 📛  which does not make sense, instead let's add its relationship with the likes table: `has_many :likes` ✔️

Reply:
Thank you for pointing out this mistake